packages/docs/src/routes/docs/components/context/index.mdx

```diff
@@ -68,27 +92,38 @@ Let's dig into every API involved:
 
 ### `createContext()`
 
-This method takes a string that gives the context unique name, if your application uses a lot of different context or you are writing a component library, we recommend to follow a name convention that avoids conflicts, such as:
+This method is used to create new `Context`.
 
 ```tsx
-export const QwikCityContext = createContext('io.builder.qwik.city');
+export interface TypedContext {
+  ...
+}
+
+export const QwikCityContext = createContext<TypedContext>('io.builder.qwik.city');
```

Comment:
I think we should show the signature of `createContext` function, so later `name` will have a reference 

like:

```
createContext<GenericType>(name: string): Context<GenericType>
```

Reply:
perfect